src/client/components/stage-details/stage-details-subcomponents/value-input.component.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ValueInput({
```

Comment:
These components still don't reveal any information about what they do and what they contain.
What is parameter and parameters?
What is level, handleValueInput?
What title, what value, what name?
What is ValueInput?

Reply:
Would it be correct to say that PARAMETERS is something like cropEnvironmentData?

Reply:
Or cropEnvironmentSensors maybe